loading.bs

```diff
@@ -147,7 +147,7 @@ spec: draft-thomson-http-mice; urlPrefix: https://tools.ietf.org/html/draft-thom
     type: dfn
         text: mi-sha256 parameter; url: section-3.1
         text: integrity proof for the first record; url: section-2.2
-spec: draft-yasskin-wpack-bundled-exchanges; urlPrefix: https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#
+spec: draft-yasskin-wpack-bundled-exchanges; urlPrefix: https://wpack-wg.github.io/bundled-responses/draft-ietf-wpack-bundled-responses.html#
```

Comment:
Can we change `spec: draft-yasskin-wpack-bundled-exchanges` to `spec: draft-ietf-wpack-bundled-responses`?

Reply:
Sure. Done.